test/run_portability_test.py

```diff
@@ -177,11 +177,10 @@ def monitor_tests(test_ids):
 
     for test_name, test_id in test_ids.items():
 
-        if terminal_tests[test_name] != "Succeeded":
```

Comment:
Is there a benefit to producing output if the build passes? It seems to me that this adds unnecessary noise to the build output.

Reply:
It was a user request. I think there are cases where the job technically succeeds, but the developer would still like to sniff around the logs to see what exactly happened.